src/bentoml/_internal/cloud/deployment.py

```diff
@@ -73,26 +73,25 @@ class DeploymentConfigParameters:
     def verify(
         self,
     ):
-        deploy_by_param = (
-            self.name
-            or self.bento
```

Comment:
Removing self.name here

Reply:
Are we removing `self.name` or `self.bento`?

Reply:
Sorry, I meant removing self.bento

Reply:
Why we need to remove `self.bento` here?

Reply:
Otherwise it would be rejected by the check:

```python
if (
            (self.config_dict and self.config_file)
            or (self.config_dict and deploy_by_param)
            or (self.config_file and deploy_by_param)
        ):
            raise BentoMLException(
                "Configure a deployment can only use one of the following: config_dict, config_file, or the other parameters"
```

Reply:
Because `bentoml deploy .` will consider `.` as the bento and this check evaluates to true and then it will skip the file argument